data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqssource/config/QueuesOptions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.sqssource.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Duration;
+import java.util.List;
+
+public class QueuesOptions {
+
+    private List<String> urls;
```

Comment:
Can we annotate this with `@JsonProperty`?

Reply:
Thanks @asifsmohammed  for your review, As Suggested @JsonProperty annotation added, but generally it will work incase both keys are matching.